### ISSUE TITLE
serverless logs should only load future logs, not logs from yesterday by default

### ISSUE THREAD

**Issue by achadee** [serverless/serverless#4438]
Hi,

Im getting a bit annoyed that the serverless logs -f -t command loads previous logs. I know that you can pass a timestamp parameter, but I shouldn't have to do it every time I want to run a log command to check my work

**Comment by horike37** [comment]
Thanks @achadee for opening :+1: 
Do you have any ideas to resolve your problem?

**Comment by achadee** [comment]
I found the test case:

https://github.com/serverless/serverless/blob/247132c97961910a07e41ba1b6e1d7e1568c0296/lib/plugins/aws/metrics/awsMetrics.test.js#L87

it looks like you are loading logs from yesterday

**Comment by achadee** [comment]
here is the line that needs to be changed, can we make it from today to today... or will that not work?

https://github.com/serverless/serverless/blob/cc94945f8e77259c7c99f865b703ad9a273c0f7b/lib/plugins/aws/metrics/awsMetrics.js#L29

**Comment by achadee** [comment]
I'm happy to open a pull request, once someone confirms this is a viable fix?

**Comment by HyperBrain** [comment]
@achadee To have a consistent behavior, we could use T - 24 hours, so that it would always show entries from up to 24 hours ago. Otherwise it might look weird if you're working late 😄 .
Do you agree?

**Comment by achadee** [comment]
@HyperBrain The problem is when I want to view the future logs; When I need to run a test with production data and the function pushes a lot of data throughout the day I have to wait up to 5 minutes for the logs to display what is happening at present time. I think if you pass the `-t` command it shouldn't show any logs until the next time the function is run or only show the last 10 lines. 

Using a timestamp to load recent logs is dangerous because it is variable to how busy the function is.

**Comment by HyperBrain** [comment]
Yes, you're right. In general I think, that the tail should behave exactly as a system `tail -f xxxx`, which is like the last 10 lines as you mentioned above. So that should be the way to go here.

**Comment by AndrewFarley** [comment]
+1 on what @HyperBrain said.  I personally can't stand using serverless logs tail feature exactly because of this.  It was so annoying that I had to write my own Python helper based on cwlogs to accomplish the same thing.  Would LOVE when follow is enabled to only show the last few lines, or if we have to keep this simple even NO logs except for anything after now would be fine (eg: setting the timestamp to NOW by default if using follow)

**Comment by brianneisler** [comment]
@horike37 @HyperBrain I think this PR may have broken log tailing. I believe the issue is that the duration it takes logs to show up in cloud watch is greater than the duration being polled for in the interval code.
